### ISSUE TITLE
✨ Update admin dashboard for communities

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#10442]
## ✨ Feature

Describe the feature that is being requested

## 🕵️ Details

Add any additional details that could assist with the development of the new feature.

## 🎨 Design File

Include a link to the design file (if it exists).

### 🧑‍🎨 Designer

Include the author of the linked design file's name (or even better, their `@githubUsername`) for future reference.

## 📸 Screenshot

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/1c830317-a14d-4463-b770-fb6d9135d640)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/16977254/98b994ae-a81b-405b-894c-8f892f4d61cc)

## 🙋‍♀️ Proposed Implementation

(optional) Do you have a proposed implementation?

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] Links on dashboard updated
- [ ] Menu updated

## 🛑 Blockers

Issues which must be completed before this one.

```[tasklist]
### Blocked By

```

**Comment by tristan-orourke** [comment]
I think I can combine this with #10368 and other tasks.

**Comment by petertgiles** [comment]
Adding buttons to the dashboard and menu was added to #10361.

**Comment by tristan-orourke** [comment]
Closing in favour of #10368 and #10361
